minoc/src/intTest/java/org/opencadc/minoc/MinocIntTest.java

```diff
@@ -116,84 +115,16 @@ public MinocIntTest() {
         RegistryClient regClient = new RegistryClient();
         anonURL = regClient.getServiceURL(MINOC_SERVICE_ID, MINOC_STANDARD_ID, AuthMethod.ANON);
         log.info("anonURL: " + anonURL);
+        certURL = regClient.getServiceURL(MINOC_SERVICE_ID, MINOC_STANDARD_ID, AuthMethod.CERT);
+        log.info("certURL: " + certURL);
+        anonSubject = AuthenticationUtil.getAnonSubject();
+        File cert = FileUtil.getFileFromResource("x509_CADCRegtest1.pem", MinocIntTest.class);
```

Comment:
We should hard code a neutral name like minoc-test.pem here; developers will have to make a symlink or copy of the cert they want to use.